docs/layout.md

```diff
@@ -0,0 +1,899 @@
+        - [ ] Prerequisites
+        - [ ] Step 1/3. Start Grafna
+        - [ ] Step 2/3. Install and configure Teleport
+        - [ ] Step 3/3. Access the application
+        - [ ] Next steps
+    - [ ] Guides
+        - w/ Tile landing
+        - [ ] Connecting Web Apps
+            - [ ] Connecting Web Applications
+            - [ ] Start Auth/Proxy service
+            - [ ] Start application service with CLI
+            - [ ] Start application service with a config file
+            - [ ] Advanced options
+            - [ ] View applications in Teleport
+            - [ ] Logging out of applications
+        - [ ] Integrating with JWTs
+            - [ ] Integrating with JWTs
+            - [ ] Introduction to JWTs
+            - [ ] Validate JWT
+        - [ ] Application API Access
+            - [ ] Application API Acces
+            - [ ] Prequisites
+            - [ ] Accessing the API
+            - [ ] Application information
+    - [ ] Access Controls
+        - [ ] Assinging labels to applications
+        - [ ] Configuring application labels in roles
+        - [ ] Integrating with identity providers
+        - [ ] Next steps
+    - [ ] Reference
+        - [ ] Configuration
+        - [ ] CLI
+- [ ] Server Access Section
+    - [ ] Introduction
+        - [ ] Server Access
+        - [ ] Demo
+        - [ ] Getting started
+        - [ ] Guides
+        - w/ Demo video
+        - w/ Tile landing
+    - [ ] Getting Started
+        - [ ] Getting Started
+        - [ ] Prerequisites
+        - [ ] Step 1/4. Create a cluster
+        - [ ] Step 2/4. Add a Node to the cluster
+        - [ ] Step 3/4. SSH into the server
+        - [ ] Step 4/4. Use tsh and the unified resource catalog
+        - [ ] Conclusion
+        - [ ] Next steps
+        - [ ] Resources
+        - w/ Diagrams
+        - w/ Screenshots
+    - [ ] Guides
+        - [ ] Using Teleport with PAM
+            - [ ] Introduction to Pluggable Authentication Modules
+            - [ ] Set up PAM on a Linux Machine running Teleport
+            - [ ] Set Message of the Day (motd) with Teleport
+            - [ ] Custom environment variables
+            - [ ] Create local users with Teleport
+            - [ ] Additional authentication steps
+        - [ ] OpenSSH Guide
+            - [ ] Overview
+            - [ ] Set up OpenSSH recording proxy mode
+            - [ ] Use OpenSSH client
+            - [ ] OpenSSH rate limiting
+            - [ ] Revoke an SSH certificate
+- [ ] Kubernetes Access Section
+    - [ ] Introduction
+        - [ ] SSO and Audit for Kubernetes Clusters
+        - [ ] Getting started
+        - [ ] Guides
+        - w/ Demo video
+        - w/ Tile landing
+    - [ ] Getting Started
+        - [ ] Getting Started
+        - [ ] Follow along with our video guide
+        - [ ] Prerequisites
+        - [ ] Step 1/3. Install Teleport
+        - [ ] Step 2/3. Create a local admin
+        - [ ] Step 3/3. SSO for Kubernetes
```

Comment:
Some of the subsections here can be put into an include and reused from the Access Controls.